code/kit_version.R

```diff
@@ -0,0 +1,107 @@
+# SET UP
+setwd("C:/Users/kitsc/KIT_code_testing/data")
+library(haven)
+library(tidyverse)
+library(plm)
+library(dplyr)
+
+# DEFINITIONS
+
+##	seed
+	seed=51
+	set.seed(seed)
+##	number of repetitions
+	reps=5000 
+##	sd of error term
+	sd = 3000
+##	coefficients on temperature term (hypothetical linear response)
+	beta = 0.25
+##	coefficients on long-run temperature interaction term
+	gamma = 0.02
+##	constant term
+	alpha = 12000
+##	difference in pixel temperature and polynomial means
+	pixel_meantemp_diff = 20
+
+# REAL COUNTRY-LEVEL TEMP DATA
+
+##	importing
+	real_temps <- read_dta("Europe_temps.dta")
+	d <- real_temps
+##	create country ID
+	d$group <- d %>% group_indices(country) 
```

Comment:
This is a small change, but since you're using `dplyr` it's nice to be consistent and make full use of the functionality it offers, instead of mixing `dplyr` and base R. I'll explain more when we talk :) 
```suggestion
	d %<>% mutate(group = group_indices(country))
```